docs/pages/management/export-audit-events/datadog.mdx

```diff
@@ -0,0 +1,417 @@
+---
+title: Monitor Teleport Audit Events with Datadog
+description: How to configure the Teleport Event Handler plugin and Fluentd to send audit logs to Datadog
+---
+
+Datadog is a SAAS monitoring and security platform. In this guide, we'll explain
+how to forward Teleport audit events to Datadog using Fluentd.
+
+The Teleport Event Handler is designed to communicate with Fluentd using mTLS
+to establish a secure channel. While we can use this same configuration to connect
+directly to [Splunk](./splunk.mdx) or [Elastic](./elastic-stack.mdx), Datadog uses
+an API endpoint with keys, so we will use Fluentd as an intermediary.
+
+{/*
+The Graph below is made with this mermaidjs logic:
+graph LR
+style e fill: #a379ff,color: #ffffff
+style f fill: #a379ff,color: #ffffff
+style c fill: #a379ff,color: #ffffff
+style g fill: #a379ff,color: #ffffff
+style i fill: #a379ff,color: #ffffff
+style e fill: #844cff,color: #ffffff
+style monitor fill:#a8afb1,stroke: #000000
+
+a(User)
+a-->e
+
+subgraph local [Local Network]
+    style local fill:#C0C0C0,stroke: #000000
+    direction LR
+    subgraph teleport [ ]
+        direction TB
+        style teleport fill:none,stroke:none
+            e(Teleport Proxy)
+        e<-->f(Teleport Auth)
+        e<-->g(Teleport Service)
+        e<-->i(Teleport Service)
+        g<-->j([Infrastructure])
+        i<-->k([Infrastructure])
+    end
+
+    subgraph monitor [Monitoring]
+        c(teleport-event-handler)
+        d(fluentd)
+    end
+    c-->d
+end
+e-->c
+d-->h(Datadog)
+*/}
+
+![Flowchart of Teleport with events forwarded to Datadog](../../../img/management/datadog-flowchart.png)
+
+## Prereqs
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- A [Datadog](https://www.datadoghq.com/) account.
+- Fluentd version v(=fluentd.version=) or greater. The Teleport Event Handler
```

Comment:
>  Is there a recommended installation method for following this guide?
I went back and forth on how to phrase this in the original writing. I think we want to avoid telling readers the best way to set up 3rd party systems in unknown environments. I'm adding a line that the guide assumes a local docker container for testing.

> Also, do we expect Fluentd to be listening on a specific port for this guide to work?

Yes, and the config file we generate and integrate in the steps below ensures that this port is listening.